tests/c_languages/cppcheck_test_files/good_file.cpp

```diff
@@ -1,5 +1,11 @@
+#include "warn_file.cpp"
 using namespace std;
-int main() {
-    cout << "Hello, world!" << endl;
+
+int main()
+  {
+    int a;
+    a = f(3,5);
+    cout << a <<endl;
     return 0;
-}
+  }
+
```

Comment:
I'm still confused why you change this, really create new test files, even though they are similar so we can be sure we don't have regressions.